### PR TITLE
github: template: Add ui-corrections

### PR DESCRIPTION
**Pull request by patrickelectric**
Github issue template for UI/UX requests

### CODE REVIEW

**Review by rafaellehmkuhl — APPROVED** [review]
Should we make exclusive for UI/UX or also allow backend problems? Will the users report backend problems?

File: .github/ISSUE_TEMPLATE/ui-corrections.yml
Comment (Williangalvani) [inline .github/ISSUE_TEMPLATE/ui-corrections.yml:9]:
```suggestion
        Thanks for taking the time to suggest new changes to our interface! Please fill out the form below to provide the necessary information for us.
```

File: .github/ISSUE_TEMPLATE/ui-corrections.yml
Comment (Williangalvani) [inline .github/ISSUE_TEMPLATE/ui-corrections.yml:14]:
```suggestion
      description: Describe what you see, why it is bad, and what you would like to see instead.
```

**Review by Williangalvani — APPROVED** [review]

**Comment by patrickelectric** [comment]
I'll wait for @ES-Alexander review before merging

**Review by ES-Alexander — CHANGES_REQUESTED** [review]
I don't really know how the structure of these works/operates, so my comments are only on the language and reasoning used. Happy for this to be merged once the suggested (or similar) clarifications have been implemented :-)

I expect this and similar templates will be quite useful for people submitting issues/feedback. Likely the largest remaining barrier to people submitting them is needing to have a GitHub account to do so - it's perhaps worth making a submission option that doesn't require logging in (maybe via [a form that automatically has its results turned into a GitHub issue](https://softwareengineering.stackexchange.com/a/339334)?). Of course if that's anonymous then we can't contact them about it if we have follow-up questions, or respond to them to let them know it's in progress/has been completed, but better than nothing. If we do make a form like that we should probably include an optional 'email' field in case they're ok with being asked for extra details, and maybe a box they can check if they would like to know when it's done or something.

In an ideal world we'd be able to set up some API stuff that can auto-send a "issue has been closed" email without needing to make their email public in the issue, and in an even more ideal world we could make comments/responses work via email (still without the user needing a GitHub account), but I expect that's likely too painful and problematic to implement.

**Comment by patrickelectric** [comment]
> I don't really know how the structure of these works/operates, so my comments are only on the language and reasoning used. Happy for this to be merged once the suggested (or similar) clarifications have been implemented :-)
> 
> I expect this and similar templates will be quite useful for people submitting issues/feedback. Likely the largest remaining barrier to people submitting them is needing to have a GitHub account to do so - it's perhaps worth making a submission option that doesn't require logging in (maybe via [a form that automatically has its results turned into a GitHub issue](https://softwareengineering.stackexchange.com/a/339334)?). Of course if that's anonymous then we can't contact them about it if we have follow-up questions, or respond to them to let them know it's in progress/has been completed, but better than nothing. If we do make a form like that we should probably include an optional 'email' field in case they're ok with being asked for extra details, and maybe a box they can check if they would like to know when it's done or something.
> 
> In an ideal world we'd be able to set up some API stuff that can auto-send a "issue has been closed" email without needing to make their email public in the issue, and in an even more ideal world we could make comments/responses work via email (still without the user needing a GitHub account), but I expect that's likely too painful and problematic to implement.

We plan to have such thing in the future, sadly, there is no time to create it right now. We have in the frontend 3 options, github, forum or our own slack (for internal usage). But thanks for the google form reference, it's a pretty neat trick!

**Review by ES-Alexander — APPROVED** [review]

**Comment by ES-Alexander** [comment]
Seems like I don't have merge permissions, but happy for this to be merged :-)

**Pull request merged**
